filebeat/Makefile

```diff
@@ -7,6 +7,10 @@ TEST_ENVIRONMENT?=true
 GOX_FLAGS=-arch="amd64 386 arm ppc64 ppc64le"
 ES_BEATS?=..
 
+TARGETS?="windows/amd64 windows/386 darwin/amd64"
+TARGETS_OLD?="linux/amd64 linux/386"
+CGO=true
```

Comment:
My only argument would be that if it breaks somewhere we know earlier. I see the community beats complication but would hope they use our packaging scripts, so they get it out of the box?